templates/definition/charger/zaptec-go2.yaml

```diff
@@ -0,0 +1,35 @@
+template: zaptec-go2
+products:
+  - brand: Zaptec
+    description:
+      generic: Go 2
+capabilities: ["rfid", "1p3p"]
+requirements:
+  evcc: ["sponsorship"]
+  description:
+    de: |
+      Im Zaptec-Portal (Installationen -> Einstellungen -> Leistungsregelung) müssen für die Zaptec Go 2 folgende Einstellungen vorgenommen werden:
+      <ul>
```

Comment:
@naltatis I dont think we can use html here?

Reply:
But it will look great.. 

<img width="815" alt="image" src="https://github.com/user-attachments/assets/8ce56138-921b-47cc-a5f4-11913618d750" />

Reply:
ping @naltatis